packages/fints/src/client.ts

```diff
@@ -92,39 +106,60 @@ export abstract class Client {
      *
      * @return A list of all statements in the specified range.
      */
-    public async statements(account: SEPAAccount, startDate: Date, endDate: Date): Promise<Statement[]> {
+    public async statements(account: SEPAAccount, startDate?: Date, endDate?: Date): Promise<Statement[]> {
         const dialog = this.createDialog();
         await dialog.sync();
         await dialog.init();
+        const segments: Segment<any>[] = [];
+        segments.push(new HKKAZ({
```

Comment:
I have no insights into the domain of this, can you link to me some documentation?